kolibri/auth/management/commands/fullfacilitysync.py

```diff
@@ -1,28 +1,32 @@
+import requests
 from django.utils.six.moves import input
+from kolibri.auth.constants.morango_scope_definitions import FULL_FACILITY
 from kolibri.auth.models import FacilityUser
-from kolibri.core.device.utils import device_provisioned
 from kolibri.core.device.models import DevicePermissions, DeviceSettings
+from kolibri.core.device.utils import device_provisioned
 from kolibri.tasks.management.commands.base import AsyncCommand
-from morango.controller import MorangoProfileController
 from morango.certificates import Certificate, Filter
+from morango.controller import MorangoProfileController
+
 
 class Command(AsyncCommand):
 
     def add_arguments(self, parser):
         parser.add_argument('--dataset-id', type=str)
         parser.add_argument('--no-push', type=bool, default=False)
         parser.add_argument('--no-pull', type=bool, default=False)
-        parser.add_argument('--host', type=str)
+        parser.add_argument('--base-url', type=str)
         parser.add_argument('--username', type=str)
         parser.add_argument('--password', type=str)
 
-    def handle_async(self, *args, **options):
+    def handle_async(self, *args, **options):  # noqa: max-complexity=16
         controller = MorangoProfileController('facilitydata')
         with self.start_progress(total=5) as progress_update:
-            network_connection = controller.create_network_connection(options['host'])
+            network_connection = controller.create_network_connection(options['base_url'])
             progress_update(1)
+
             # get list of facilities and if more than 1, display all choices to user
-            facility_resp = network_connection._request('api/facility/')
+            facility_resp = requests.get(options['base_url'] + 'api/facility/')
```

Comment:
Please create tech debt issue: use `urljoin` and make sure it's tolerant to present or missing trailing slash.

Reply:
Also, why not leverage ` network_connection._request`?

Reply:
Internal function?
I suppose I don't see any harm in making it public. just wrote it in mind for using our morango api endpoints, but in reality you can use it on any api endpoints.